Sample started code.py

```diff
@@ -1,54 +0,0 @@
-# -*- coding: utf-8 -*-
-import pandas as pd
-import numpy as np
-import MultiPyVu as mpv
-
-keithley = Keithley2400("GPIB::xxxx")

```

Comment:
This is the right idea but where are you getting this Keithley2400 class? I don't see an import. Please check out this repo where I've implemented a class like this for the 2400: https://github.com/Quantum-Science-and-Technology-Lab/labber-wrapper.

Reply:
One caveat is that this code relies on Labber. I don't know if it is installed on the PPMS computer. Maybe we can install it.